src/elements/content-sidebar/activity-feed/task-new/AssigneeList.scss

```diff
@@ -13,20 +13,17 @@ $avatar-size: 28px;
     display: flex;
     margin: 6px 0 0 0;
     padding: 3px 0;
+}
 
-    .bcs-AssigneeList-detailsStatus {
-        color: $bdl-neutral-02;
-        font-size: 12px;
-    }
+.bcs-AssigneeList-detailsStatus {
+    color: $bdl-neutral-02;
```

Comment:
We could swap in the `bdl-gray` equivalent here, but that's an easy find/replace that can be done later.

Reply:
we were using the neutral scheme pretty consistently for tasks features, so let's commit to that in a larger scrub later